components/form/formitems/overview.md

```diff
@@ -16,6 +16,7 @@ In this article:
 
 * [Basics](#basics)
 * [Example - Customize the appearance of the editors in the Form](#customize-the-appearance-of-the-editors-in-the-form)
+* [Hybrid Form - Combination of manually defined and autogenerated fields](#hybrid-form---combination-of-manually-defined-and-autogenerated-fields)
```

Comment:
Is "hybrid" an official term that we are introducing and will use everywhere (docs, demos, support)? If yes, OK. If not, then better not introduce it at all.